components/src/preact/shared/stories/expectInvalidAttributesErrorMessage.ts

```diff
@@ -0,0 +1,13 @@
+import { expect, waitFor, within } from '@storybook/test';
+
+export async function expectInvalidAttributesErrorMessage(canvasElement: HTMLElement, errorMessage: string) {
+    const canvas = within(canvasElement);
+
+    await waitFor(() =>
+        expect(canvas.getByText('Error - Invalid component attributes', { exact: false })).toBeInTheDocument(),
+    );
+
+    await canvas.getByRole('button', { name: 'Show details.' }).click();
```

Comment:
My IDE claims that this doesn't need to be awaited
```suggestion
    canvas.getByRole('button', { name: 'Show details.' }).click();
```

Reply:
True. Mine too :)